UPGRADING.md

```diff
@@ -0,0 +1,89 @@
+# Upgrading Your Chain to Use FeeMarket
+
+## Code Changes
+
+To integrate your chain with `x/feemarket`, the following steps should be performed:
+
+### Add the Module to Your App
+
+- The `FeeMarketKeeper` must be added to your application as seen [here](https://github.com/skip-mev/feemarket/blob/0f83e172c92a02db45f83bf89065fd9543967729/tests/app/app.go#L163).
+- A `DenomResolver` (if desired) must be set in your application as seen [here](https://github.com/skip-mev/feemarket/blob/0f83e172c92a02db45f83bf89065fd9543967729/tests/app/app.go#L509).
+- `Ante` and `Post` handlers must be configured and set with the application `FeeMarketKeeper` as seen [here](https://github.com/skip-mev/feemarket/blob/0f83e172c92a02db45f83bf89065fd9543967729/tests/app/app.go#L513).
+
+### Determine Parameters
+
+We provide sensible default parameters for running either the [EIP-1559](https://github.com/skip-mev/feemarket/blob/0f83e172c92a02db45f83bf89065fd9543967729/x/feemarket/types/eip1559.go#L56) or [AIMD EIP-1559](https://github.com/skip-mev/feemarket/blob/0f83e172c92a02db45f83bf89065fd9543967729/x/feemarket/types/eip1559_aimd.go#L65) feemarkets. 
+
+> **Note**
+>
+> The default parameters use the default Cosmos SDK bond denomination. The should be modified to your chain's fee denomination.
+
+## Changes for End-Users
```

Comment:
We should create a separate doc here for "How to Integrate x/feemarket as an end user"  then just link it here

Reply:
You could leave the enumeration of the changes here, but you should move the next section to a new doc. 

The new doc should just be "How to set fees for chains that use x/feemarket" -- just get right into it. Don't give historical context of how things are now different or how they used to be